ga4gh/_protocol_definitions.py

```diff
@@ -14,6 +14,107 @@
 version = '0.6.0a1'
 
 
+class BioSample(ProtocolElement):
+    """
+    A BioSample refers to a unit of biological material from which the
+    substrate    molecules (e.g. genomic DNA, RNA, proteins) for
+    molecular analyses (e.g.    sequencing, array hybridisation, mass-
+    spectrometry) are extracted. Examples    would be a tissue biopsy,
+    a single cell from a culture for single cell genome    sequencing
+    or a protein fraction from a gradient centrifugation.    Several
+    instances (e.g. technical replicates) or types of experiments
+    (e.g.    genomic array as well as RNA-seq experiments) may refer
+    to the same BioSample.    In the context of the GA4GH metadata
+    schema, BioSample constitutes the central    reference object.
+    """
+    _schemaSource = """
+{"namespace": "org.ga4gh.models", "type": "record", "name":
+"BioSample", "fields": [{"doc": "", "type": "string", "name": "id"},
+{"default": null, "doc": "", "type": ["null", "string"], "name":
+"name"}, {"default": null, "doc": "", "type": ["null", "string"],
+"name": "description"}, {"doc": "", "type": "string", "name":
+"created"}, {"doc": "", "type": "string", "name": "updated"},
+{"default": null, "doc": "", "type": ["null", "string"], "name":
+"individualId"}, {"default": null, "doc": "", "type": ["null",
+"string"], "name": "collected"}, {"default": {}, "doc": "", "type":
+{"values": {"items": "string", "type": "array"}, "type": "map"},
+"name": "info"}], "doc": ""}
+"""
```

Comment:
this is hard to read.  It also creates a maintaince headache having the schema in two places.